app.js

```diff
@@ -20,6 +21,53 @@ const auth = authorization(app);
 app.use(auth.initialize());
 app.auth = auth;
 
+let detailsRoute = function(req, res, next) {
+    console.log(req.method + " " + req.url);
+    
+    if (req.headers && req.headers.authorization) {
+        console.log("Token: " + req.headers.authorization.substring(4, req.headers.authorization.lenght));
+    }
+
+    let token = req.headers.authorization ? req.headers.authorization.substring(4, req.headers.authorization.lenght).trim() : undefined;
+
+    if (token) {
+        let tokenDecode = jwt.decode(token, config.jwtSecret);
+        let Users = app.datasource.models.Users;
+        
+        Users.findOne({ where: { id: tokenDecode.id } }).then((user) => {
```

Comment:
You shouldn't need `findOne`, is there a function called `findBy` or `findById`?